pubsub/README.md

```diff
@@ -67,8 +67,10 @@ You can find information about the PubSub research and notes in the following re
   - [libp2p/go-libp2p-pubsub/floodsub.go](https://github.com/libp2p/go-libp2p-pubsub/blob/master/floodsub.go);
   - [libp2p/js-libp2p-floodsub](http://github.com/libp2p/js-libp2p-floodsub);
   - [libp2p/rust-libp2p/floodsub](https://github.com/libp2p/rust-libp2p/tree/master/protocols/floodsub)
+  - [status-im/nim-libp2p/floodsub](https://github.com/status-im/nim-libp2p/blob/master/libp2p/protocols/pubsub/floodsub.nim)
 - GossipSub, extensible baseline pubsub (2018)
   - [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub#implementation-status)
+  - [status-im/nim-libp2p/gossipsub](https://github.com/status-im/nim-libp2p/blob/master/libp2p/protocols/pubsub/gossipsub.nim)
```

Comment:
This should be in the gossipsub page, not the top level spec.

Reply:
Moved